File: dayTwo/Java/Level1/TaskTen.java
import java.util.Scanner;

public class TaskTen {

    public static void main(String[] args) {

        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter first number: ");
        double first_Number = userInput.nextDouble();
         System.out.print("Enter second number: ");
         double second_Number = userInput.nextDouble();
         System.out.print("Enter third number: ");
         double third_Number = userInput.nextDouble();
        double average = (first_Number + second_Number + third_Number) / 3;
         System.out.println("Average: " + average);
        
    }
}

File: dayTwo/JavaScript/Level1/TaskTen.js
const prompt = require('prompt-sync')();
let firstNumber = Number(prompt("Enter first number: "));
let secondNumber = Number(prompt("Enter second number: "));
let thirdNumber= Number(prompt("Enter third number: "));
let average = (firstNumber + secondNumber + thirdNumber) / 3;

console.log(`The average is ${average}`);

